tools/native/src/main/scala/scala/scalanative/build/Platform.scala

```diff
@@ -42,6 +42,13 @@ object Platform {
    */
   lazy val isFreeBSD: Boolean = LinktimeInfo.isFreeBSD
 
+  /** Test for the platform type
+   *
+   *  @return
+   *    true if `OpenBSD`, false otherwise
+   */
+  lazy val isOpenBSD: Boolean = LinktimeInfo.isOpenBSD
```

Comment:
You forget to add entry in `scala.scalanative.meta.LinktimeInfo`. Then you should also populate predefied linktime resolver entry in https://github.com/scala-native/scala-native/blob/main/tools/src/main/scala/scala/scalanative/linker/LinktimeValueResolver.scala#L17 using a value from `Config.targetsOpenBSD`

Reply:
@WojciechMazur are you sure that the last one needed? I don't see FreeBSD related settings inside `LinktimeValueResolver`.

Reply:
Oh, sorry, you're right. If the `isOpenBSD` is linktime resolved (is calculated based on `target.os` like freeBSD then it's not needed.